packages/apex-node/src/tests/testService.ts

```diff
@@ -185,6 +185,7 @@ export class TestService {
    * Asynchronous Test Runs
    * @param options test options
    * @param codeCoverage should report code coverage
+   * @param exitOnTestRunId should not wait for test run to complete, return test run id immediately
```

Comment:
nit: not new I realize but this parameter name is kind of confusing.  Could we go with `immediatelyReturn` or `runInBackground`

Reply:
Mmmm maybe - it's in several functions, so update in all places?

Reply:
eh..I don't want to scope creep you.  This is the entry point so def here, but if it _feels_ right to update down the chain feel free.

Reply:
Done - and renamed the function for clarity too.